import/marc_local.properties

```diff
@@ -87,3 +87,14 @@
 # library or repository systems and can be a useful match point with third party
 # systems.
 #uuid_str_mv = 024$a ? (ind1 == 7 && ($2 == 'uuid' || $2 == 'UUID')
+
+# Uncomment the following lines to use SolrMarc plug-in for indexing MARC-based hierarchies
+#hierarchytype = custom(org.vufind.Collection), getHierarchyType
+#hierarchy_top_id = custom(org.vufind.Collection), getHierarchyTopID
+#hierarchy_top_title = custom(org.vufind.Collection), getHierarchyTopTitle
+#hierarchy_parent_id = custom(org.vufind.Collection), getParentID
+#hierarchy_parent_title = custom(org.vufind.Collection), getParentTitle
+#hierarchy_sequence = custom(org.vufind.Collection), getSequence
+#is_hierarchy_id = custom(org.vufind.Collection), isHierarchyID
+#is_hierarchy_title = custom(org.vufind.Collection),isHierarchyTitle
+#hierarchy_browse = custom(org.vufind.Collection), getHierarchyBrowse
```

Comment:
How would you feel about passing the ILS prefix as a parameter here (as alluded above), so that these end up looking like:

```
#hierarchy_top_id = custom(org.vufind.Collection), getHierarchyTopID(vtls)
#hierarchy_top_title = custom(org.vufind.Collection), getHierarchyTopTitle(vtls)
#hierarchy_parent_id = custom(org.vufind.Collection), getParentID(vtls)
#hierarchy_parent_title = custom(org.vufind.Collection), getParentTitle(vtls)
#hierarchy_sequence = custom(org.vufind.Collection), getSequence(vtls)
#is_hierarchy_id = custom(org.vufind.Collection), isHierarchyID(vtls)
#is_hierarchy_title = custom(org.vufind.Collection),isHierarchyTitle(vtls)
#hierarchy_browse = custom(org.vufind.Collection), getHierarchyBrowse(vtls)
```

I realize it's a bit redundant, but it's also more specific. Implementing this would just require adding a string argument to all the methods in the class, and passing it down to support methods as needed.